CaliptraChecklistAndEvaluationMethodology.md

```diff
@@ -0,0 +1,303 @@
+  * **Evaluation Methodology:** Manufacturers MUST provide a state machine diagram or equivalent documentation showing how Caliptra's mode transitions are managed and synchronized with the SoC's states.
+
+### *Error Handling*
+
+* **Checklist Item:**  
+  * **Requirement:** The SoC MUST correctly implement error handling and reporting mechanisms for interactions with Caliptra, especially for security-relevant events, without leaking sensitive information.  
+  * **Evaluation Methodology:** Manufacturers MUST document the error handling procedures, including what information is reported, how errors are logged, and measures taken to prevent information leakage. Specific error codes and messages should be reviewed for appropriateness.
+
+### *Log Functionality in SoC*
+
+* **Checklist Item:**  
+  * **Requirement:** The SoC SHOULD implement logging functionality that records relevant events related to Caliptra operations, ensuring logs are tamper-evident and provide necessary information for attestation. The log does not need to be kept secure but should be tamper-evident to detect manipulation.  
+  * **Evaluation Methodology:** Manufacturers SHOULD describe the logging mechanism, including how logs are structured, how they are protected against tampering (e.g., using cryptographic hash extensions), and how they integrate with Caliptra's measurement processes.
+
+## Secure Processes
+
+### Development Process
+
+#### *Integrity of Firmware and Hardware*
+
+* **Checklist Item:**  
+  * **Requirement:** The Caliptra RTL MUST match an official release. Netlists, layout, and other development collateral derived from the RTL MUST maintain the same behavior as the released RTL.  
+  * **Evaluation Methodology:** Manufacturers MUST provide evidence of hash verification against official releases. Manufacturers MUST provide evidence that collateral derived from the released RTL has not had its behavior modified.
+
+#### *Caliptra FW Cryptographic Hash*
+
+* **Checklist Item:**  
+  * **Requirement:** Cryptographic hashes of the Caliptra ROM and firmware (FMC and RT) MUST match Caliptra reference hashes of official Caliptra releases.  
+  * **Evaluation Methodology:** Manufacturers MUST provide evidence of hash verification against official releases.
+
+#### *Testing and Verification*
+
+* **Checklist Item:**  
+  * **Requirement:** The product SHOULD undergo thorough testing and verification during development, focusing on both functional correctness and security aspects. This includes testing of specific features and security mechanisms.  
+  * **Evaluation Methodology:** Manufacturers SHOULD present test plans, results, and security analyses covering critical functionalities such as:  
```

Comment:
Just a question: I have seen test cases in the Caliptra RTL code base as well as FW code base. Shouldn't they be applied?